Gemfile

```diff
@@ -1,22 +1,3 @@
 source 'https://rubygems.org'
 
 gemspec
-
```

Comment:
I've actually answered my own question here after reading up on it, I'll update the PR